packages/plugins/users-permissions/server/bootstrap/index.js

```diff
@@ -27,7 +27,14 @@ module.exports = async ({ strapi }) => {
   await getService('users-permissions').initialize();
 
   if (!strapi.config.get('plugin.users-permissions.jwtSecret')) {
-    const jwtSecret = uuid();
+    const jwtSecret = crypto.randomBytes(16).toString('base64');
+
+    if (process.env.NODE_ENV === 'production') {
+      throw new Error(
+        `[Users & Permissions] Missing jwtSecret. Please, in config/plugins.js, set config.jwtSecret for the users-permissions plugin or set environment variable JWT_SECRET (ex: ${jwtSecret}).`
```

Comment:
We shouldn't tell them to statically set the key in this file, they should use the environment variable generally.

This message is better than the other two though

Reply:
It doesn't work the same for this key, that's why. I'll change the message